catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonCatalogOperations.java

```diff
@@ -241,7 +276,63 @@ public PaimonTable createTable(
       SortOrder[] sortOrders,
       Index[] indexes)
       throws NoSuchSchemaException, TableAlreadyExistsException {
-    throw new UnsupportedOperationException();
+    NameIdentifier nameIdentifier = buildPaimonNameIdentifier(identifier);
+    NameIdentifier schemaIdentifier = NameIdentifier.of(nameIdentifier.namespace().levels());
+    if (!schemaExists(schemaIdentifier)) {
+      throw new NoSuchSchemaException(NO_SUCH_SCHEMA_EXCEPTION, schemaIdentifier);
+    }
+    Preconditions.checkArgument(
+        partitioning == null || partitioning.length == 0,
+        "Table Partitions are not supported when creating a Paimon table in Gravitino now.");
+    Preconditions.checkArgument(
+        sortOrders == null || sortOrders.length == 0,
+        "Sort orders are not supported for Paimon in Gravitino.");
+    Preconditions.checkArgument(
+        indexes == null || indexes.length == 0,
+        "Indexes are not supported for Paimon in Gravitino.");
+    Preconditions.checkArgument(
+        distribution == null || distribution == Distributions.NONE,
```

Comment:
can specify the paimon distribution policy through table properties.

Reply:
https://paimon.apache.org/docs/master/maintenance/configurations/#coreoptions
users should set `bucket-key` in table properties

Reply:
As Gravitino provides unified interface, maybe we should transform `bucket-key` to Distribution in engine side

Reply:
ok, but i prefer to support it in another pr.